modules/ROOT/pages/app-to-app-requests.adoc

```diff
@@ -0,0 +1,23 @@
+= Access an Application in the Same Cluster
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In an Anypoint Runtime Fabric cluster, every application replica exposes an endpoint inside the cluster. When you enable inbound traffic, that endpoint is exposed externally. Refer to xref:enable-inbound-traffic.adoc[Enable Inbound Traffic on Anypoint Runtime Fabric] for additional information about servicing external requests and configuring the load balancer. 
+
+To send a request from one application to another application inside the cluster, use the application name and environment ID. 
+
+Every application has a DNS record in the cluster internal domain. For example, the DNS record for the application `hello-world` in the  `2abc2ce6-cce6-463b-afaa-a1979e3xxxxx` environment is:
```

Comment:
How does this DNS record relate to the destination environment and name? Why provide an example of the record, when the request requires a destination app. environment and app. name?